python/phoenixdb/NEWS.rst

```diff
@@ -1,3 +1,19 @@
+..
```

Comment:
According to http://www.apache.org/legal/src-headers.html#faq-exceptions README type .rst or .RST files don't need a license header. 
Since these files are meant to be read without preprocessing, the license headers are inconvenient for the reader. (I am not talking about the .rst files in the doc dir, which are meant to be processed)

Reply:
Hrm, I would normally disagree with you, but if I read, alone, "Short informational text files", I can see your interpretation of this. I'll back it out of these three files (NEWS, README, and RELEASING) and add rat check exclusions.